app/lib/core/models/loading_vehicle_info.dart

```diff
@@ -119,6 +133,25 @@ class AnimalGroup {
       : _compromisedAnimals = compromisedAnimals,
         _specialNeedsAnimals = specialNeedsAnimals;
 
+  factory AnimalGroup.defaultAnimalGroup(
```

Comment:
- [ ] There should not be a default animal group. Anything with a list of animal groups should have an empty list `[]`
> You are already doing this, so this constructor isn't being used!